### PR TITLE
Rework FAQ

### PR DESCRIPTION
**Pull request by Kyniverse**
Updated the FAQ to be more informative and "transparent"

### CODE REVIEW

File: faq.markdown [hunk 1]
```diff
@@ -4,79 +4,108 @@ title: FAQ
...
 
+# How many computers can I install and use the plugin on?
+You can install and use our plugins on as many computers are you like.
 
-# Is an internet connection required?
-Once downloaded, all Darkpalace Studio plugins run completely offline.
+
+
+# Do the plugins require an internet connection to function?
+No, all of your plugins *run completely offline* and do not require any form of internet
```
Comment (Laguna1989) [inline faq.markdown:31]: I am not quite sure about the inconsistent usage of 
- "our" plugins
- "your" plugins 
- "the" plugins
- Darkpalace Studio plugins

No strong preference for any of those, I would just like it to be consistent.

File: faq.markdown [hunk 2]
```diff
@@ -4,79 +4,108 @@ title: FAQ
...
 
-- try to "call home"
-- require any sort of online service
-- include any sort of revocation or invalidation mechanism
 
 
 # How can I purchase any of the plugins?
-Click the buy button on the plugin website. It will take you to the payment provider which handles secure payment and download of the actual files.
+All of our plugins are sold through Tentary which you can access by clicking the "Buy" button
+located top-left of the designated plugin page which will take you to the payment provider
```
Comment (Laguna1989) [inline faq.markdown:44]:
```suggestion
located top of the designated plugin page which will take you to the payment provider
```

At least for me the button is top-right, not top-left. Am I seeing an old version?

File: faq.markdown [hunk 3]
```diff
@@ -4,79 +4,108 @@ title: FAQ
...
+which handles secure payment and provides a way to download the files.
```
Comment (Laguna1989) [inline faq.markdown:45]:
```suggestion
which handles secure payment and taxes. Tentary also provides you a way to download the files.
```

File: faq.markdown [hunk 4]
```diff
@@ -4,79 +4,108 @@ title: FAQ
...
+
+You can also find all of our plugins on the [Tentary Shop](https://darkpalacestudio.tentary.com/)
 
-You can find all plugins also on the [shop](https://darkpalacestudio.tentary.com/)
 
 
 # Do you offer sales?
+We believe that our plugins are offered at a reasonable price considering the current market
```
Comment (Laguna1989) [inline faq.markdown:52]:
```suggestion
We believe that our plugins are offered at a reasonable price considering the current market.
```

File: faq.markdown [hunk 5]
```diff
@@ -4,79 +4,108 @@ title: FAQ
...
 Darkpalace Studio plugins are offered at a reasonable fixed price.
 Setting fixed prices is a very fair option for all involved parties.
 Thus there will be no sales or any bundle deals.
```
Comment (Laguna1989) [inline faq.markdown:55]:
```suggestion
Thus there will be no sales.

Note that we do offer Affiliate links where a percentage of the price is a commission. The price for the customer stays the same.
```

File: faq.markdown [hunk 6]
```diff
@@ -4,79 +4,108 @@ title: FAQ
...
-All future updates are free to owners.
+All of our plugins come with free life-time updates.
+
 
 
 # Do you offer demo versions or trials?
-No. As a small plugin developer time management is a crucial part. Building a demo version is actually a lot of effort.
-I'd rather spend that time on building great quality audio plugins and add new features.
+As of this moment, no. As a new and small plugin develop, time management is a crucial and difficult part.
+Building a demo is a lot of effort to both develop and manage and goes against our principle of DRM-protection.
```
Comment (Laguna1989) [inline faq.markdown:71]:
```suggestion
Building a demo is a lot of effort regarding development and maintenance.  Also it goes against our principle of no DRM-protection.
```

File: faq.markdown [hunk 7]
```diff
@@ -4,79 +4,108 @@ title: FAQ
...
 
 
 # Do you offer refunds?
-As the plugins contain no DRM or licensing, there is generally no refund option.
+Due to our plugins not containing any form of DRM (Digital Rights Management) we generally do not
+issue any refunds. Our reasoning is that a customer could buy one of our products, request a refund 
+and then continue to use our products.
+
+In the case that you are truly unsatisfied with one of our products feel free to get in touch so
+that we can discuss potential solutions.
```
Comment (Laguna1989) [inline faq.markdown:84]:
```suggestion
In the case there are any technical issues with one of our products, feel free to get in touch so
that we can discuss potential solutions.
```

File: faq.markdown [hunk 8]
```diff
@@ -4,79 +4,108 @@ title: FAQ
...
 # Your sales strategy is different from Y! Can you do Z?
-Most likely no. Most companies are in for the money, not for the customer.
-That is why they offer/require bundles, sales, high upgrade fees, DRM, hardware dongles, subscription models and other pieces from the [Enshittification](https://www.wired.com/story/tiktok-platforms-cory-doctorow/) toolbox. 
-At Darkpalace Studio things are handled differently. Here it is "[Buy it, own it](https://doctorow.medium.com/https-pluralistic-net-2023-12-08-playstationed-tyler-james-hill-2ba28bfdbefc)".
+Most like not, most companies are focussing a lot of time and effort into generating as much revenue 
+as possible by utilizing various strategies such as:
+- False advertisement as to what a product actually does
+- Offering bundles as 'to-good-to-be-true'
+- High upgrade fees
+- DRM (login, dongles, etc)
```
Comment (Laguna1989) [inline faq.markdown:94]:
```suggestion
- DRM (login, dongles, etc)
- Subscription based services
- Verndor lock-ins
```

File: faq.markdown [hunk 9]
```diff
@@ -4,79 +4,108 @@ title: FAQ
...
+
+And many more tools from the [Enshittification](https://www.wired.com/story/tiktok-platforms-cory-doctorow/) toolbox.
+At Darkpalace Studio we believe in the concept of "[You Buy it, You own it](https://doctorow.medium.com/https-pluralistic-net-2023-12-08-playstationed-tyler-james-hill-2ba28bfdbefc)"
+
 
 
-# Can I create my own theme for a plugin?
-Yes. All Darkpalace Studio plugins are highly customizable. Please take a look at the respective manual explaining the details.
+# Some of these themes look great but can I create my own?
```
Comment (Laguna1989) [inline faq.markdown:101]:
```suggestion
# Some of these themes look great! Can I create my own?
```

File: faq.markdown [hunk 10]
```diff
@@ -4,79 +4,108 @@ title: FAQ
...
+Yes! All of our plugins are highly customizable, take a look at the respective manual for more details.
```
Comment (Laguna1989) [inline faq.markdown:102]:
```suggestion
Yes! All of our plugins are highly customizable, take a look at the respective manual for more details and feel free to contact us if there are any questions.
```

File: faq.markdown [hunk 11]
```diff
@@ -85,33 +114,52 @@ More info can be found in
 - [this website](https://syntheway.com/fix-au-vst-vst3-macos.htm).
 
 
-# The plugin looks weird with display scaling
-Windows does not consistently forward display scaling to plugins.
 
-If you are using windows settings to scale your screen and the plugin looks wei+rd, you can fix this by changing the value `custom_ui_scaling_factor` in the theme of your plugin, e.g.`C:\Users\Public\Documents\Darkpalace Studio\sloth\themes\Dark.json`.
+# The plugin appears to look incorrectly scaled on Windows
```
Comment (Laguna1989) [inline faq.markdown:118]:
```suggestion
# The plugin appears to look incorrectly scaled
```

Might also happen on macOS or any other operating system that does weird stuff with display scaling

File: faq.markdown [hunk 12]
```diff
@@ -85,33 +114,52 @@ More info can be found in
...
+Once you’ve found this, change is to a decimal value representing your display-scaling
+value. e.g. If your display-scaling is set to 125% in windows, change the value of 
+custom_ui_scaling_factor to 1.25
+
 
-When your Windows Settings show a display scaling of 125%, you need to set `custom_ui_scaling_factor` to `1.25`
-respectively.
 
+# When running the windows installer an error pops-up regarding installing the VST (Error 183)
+Windows is known to have issues with file-locking, before running the installer please make sure
```
Comment (Laguna1989) [inline faq.markdown:138]:
```suggestion
Windows is known to have issues with file-locking. Before running the installer, please make sure
```

File: faq.markdown [hunk 13]
```diff
@@ -85,33 +114,52 @@ More info can be found in
...
+that any VST-Host applications are closed and, if present, try to delete the already existing
+plugin files.
```
Comment (Laguna1989) [inline faq.markdown:140]:
```suggestion
plugin files before installation.
```

File: faq.markdown [hunk 14]
```diff
@@ -85,33 +114,52 @@ More info can be found in
...
 
-# The windows installer complains about installing the vst3 (error 183)
-Please try to delete any old vst3 manually and run the installer again.
 
-# The plugin UI does not open (but it does not crash)
-This can be fixed by deleting the `editor.json` file in the respective folder:
+
+# When I try opening the plugin UI nothing shows up (But it does not crash)
+This is a known bug and we are working on fixing this, for now this can be fixed by going to
```
Comment (Laguna1989) [inline faq.markdown:145]:
```suggestion
This is a known issue with some DAWs. For now this can be fixed by going to
```

**Review by Laguna1989 — CHANGES_REQUESTED** [review]
Looks great and reads so much better. I added some suggestions and fixes for some typos. Feel free to directly accept them or discuss.

File: faq.markdown (hunk 1 above)
Comment (Kyniverse) [reply faq.markdown:31]: Good catch! For the requestions I will refer to "your" and for the answers I will refer to "our"

**Comment by Kyniverse** [comment]
Applied the changed and fixed some spelling mistakes. Request for another review @Laguna1989

**Review by Laguna1989 — APPROVED** [review]
Looks great

**Pull request merged**